Source/Core/EllipsoidTangentPlane.js

```diff
@@ -230,5 +314,56 @@ define([
         return result;
     };
 
+    var scratchOffset = new Cartesian3();
+    var scratchScale = new Cartesian3();
+    /**
+     * Computes an OrientedBoundingBox given extents in the east-north-up space of the tangent plane.
+     *
+     * @param {Number} minX Minimum x extent in tangent plane space.
+     * @param {Number} maxX Maximum x extent in tangent plane space.
+     * @param {Number} minY Minimum y extent in tangent plane space.
+     * @param {Number} maxY Maximum y extent in tangent plane space.
+     * @param {Number} minZ Minimum z extent in tangent plane space.
+     * @param {Number} maxZ Maximum z extent in tangent plane space.
+     * @param {OrientedBoundingBox} [result] The object onto which to store the result.
+     * @returns {OrientedBoundingBox} The modified result parameter or a new OrientedBoundingBox instance if one was not provided.
+     */
+    EllipsoidTangentPlane.prototype.extentsToOrientedBoundingBox = function(minX, maxX, minY, maxY, minZ, maxZ, result) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(minX)) { throw new DeveloperError('minX is required.'); }
```

Comment:
We don't bunch up `if` blocks like this (you could also just use the auto code formatter in Eclipse).

To make this less redundant, you could just have one `if` statement with `||`s.  This exception message will not be as precise, but that is fine since it is a rare case.